common/src/main/java/com/netflix/conductor/common/metadata/workflow/WorkflowDef.java

```diff
@@ -101,6 +101,12 @@ public enum TimeoutPolicy {
     @ProtoField(id = 15)
     private Map<String, Object> inputTemplate = new HashMap<>();
 
+    @ProtoField(id = 16)
```

Comment:
This should not be in the WorkflowDef.

The idea with the status publisher is that status publisher is called in the workflow's lifecycle and any configuration should be specific to that status publisher.